extensions/git/src/commands.ts

```diff
@@ -2488,6 +2488,33 @@ export class CommandCenter {
 		await commands.executeCommand('revealInExplorer', resourceState.resourceUri);
 	}
 
+	@command('git.revealFileInOS.linux')
+	async revealFileInOSlinux(resourceState: SourceControlResourceState): Promise<void> {
+		return this.revealFileInOS(resourceState);
+	}
+
+	@command('git.revealFileInOS.mac')
+	async revealFileInOSmac(resourceState: SourceControlResourceState): Promise<void> {
+		return this.revealFileInOS(resourceState);
+	}
+
+	@command('git.revealFileInOS.windows')
+	async revealFileInOSwindows(resourceState: SourceControlResourceState): Promise<void> {
+		return this.revealFileInOS(resourceState);
+	}
+
+	private async revealFileInOS(resourceState: SourceControlResourceState): Promise<void> {
```

Comment:
As far as I know we can add multiple decorators to a method so we should be able to do something like this:

```ts
@command('git.revealFileInOS.linux')
@command('git.revealFileInOS.mac')
@command('git.revealFileInOS.windows')
async revealFileInOS(resourceState: SourceControlResourceState): Promise<void> {
```

Reply:
Thanks for that tip. Change pushed.